### PR TITLE
Fix k6 browser type definitions

### PR DESCRIPTION
**Pull request by ankur22**
When testing the type definitions for the k6 browser module, we found that the type definitions didn't correctly reflect the usage. This change fixes this issue and ensures that the `browser` type is the entry point for working with the browser module.

Previously this is how the type definitions were defined (which is incorrect):

```js
import { Browser, Page, BrowserContext } from 'k6/experimental/browser'
...
    const browser = new Browser();
```

1. xk6-browser only export `browser`, not even `Browser`.
2. `browser` a type that we should be able to work with directly without having to new up an instance of one.

The aim of this PR is to correctly reflect the implementation of the k6 browser module (xk6-browser) APIs so that it works like so:

```js
import { browser } from 'k6/experimental/browser'
...
    const page = browser.newPage();
```

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/k6/experimental/browser.d.ts [hunk 1]
```diff
@@ -1,3 +1,5 @@
+export {};
```
Comment (ankur22) [inline types/k6/experimental/browser.d.ts:1]: This helps to enforce us to be explicit in what we export, otherwise all types are exported

**Review by inancgumus — APPROVED** [review]
👍 LGTM! We can add `devices` export later on 👍

File: types/k6/experimental/browser.d.ts (hunk 1 above)
Comment (oleiade) [reply types/k6/experimental/browser.d.ts:1]: TIL 🙇🏻

**Review by oleiade — APPROVED** [review]

**Comment by CNManning** [comment]
I don't understand the desire/need to remove all type exports from this. Changing classes from `export class X` to `export interface X` would make sense so that the classes can't be manually created via `new X`. Exports of the form `export type X...` and `export interface X...` are still useful and desirable however. Without exporting them it's not possible/easy to correctly type variables that method returns are assigned to. 

For instance without exporting the `Page` interface the following wouldn't be possible:

```
import { browser, Page } from `k6/experimental/browser';

async function doSomethingWithPage(page: Page): Promise<void> {
  await page.goto('http://www.example.com');

  // do more stuff with the page
}

export default async function () {
  const page = browser.newPage();

  await doSomethingWithPage(page);
}
```

it would instead be necessary to do the following which has no type checking due to the use of `any` as the type for the `page` parameter to `doSomethingWithPage `:

```
import { browser } from `k6/experimental/browser';

async function doSomethingWithPage(page: any): Promise<void> {
  await page.goto('http://www.example.com');

  // do more stuff with the page
}

export default async function () {
  const page = browser.newPage();

  await doSomethingWithPage(page);
}
```

**Comment by ankur22** [comment]
Hi @CNManning,

Thanks for the feedback, it's insightful and super useful!

> Changing classes from export class X to export interface X would make sense so that the classes can't be manually created via new X

Ok, that's good to know I got this correct 👍 

>  Exports of the form export type X... and export interface X... are still useful and desirable however. Without exporting them it's not possible/easy to correctly type variables that method returns are assigned to.

This is a use case I hadn't considered. I was too focused on trying to reflect type definitions against the types that are being exported by the k6 browser module. I'll revert the _unexport_ change and export the types again.

Thanks again for the valuable feedback!

**Comment by ankur22** [comment]
@CNManning I believe 07ab584f5404f8266e8bbf2367fc083f1287be01 resolves the use case you mentioned.

**Review by ka3de — APPROVED** [review]
LGTM.

**Review by mstoykov — APPROVED** [review]

**Comment by ankur22** [comment]
Ready to merge

**Pull request merged**